pandas/core/window.py

```diff
@@ -462,7 +462,8 @@ class Window(_Window):
     min_periods : int, default None
         Minimum number of observations in window required to have a value
         (otherwise result is NA). For a window that is specified by an offset,
-        this will default to 1.
+        this will default to 1. Otherwise, this will default to the size of the
+        window.
```

Comment:
Nit: the pronoun "this" is grammatically vague here.  Can we replace "this" with "min_periods" (surrounded with backticks)?